src/TestGrains/EventSourcing/PersonState.cs

```diff
@@ -27,7 +27,7 @@ MIT License
 
 namespace TestGrains
 {
-    public class PersonState : JournaledGrainState
+    public class PersonState : JournaledGrainState<PersonState>
```

Comment:
So just to make sure I understood correctly: this is an example of dynamic case? Can we then rename it to `DynamicPersonState`.